### DISCUSSION TITLE
Troubleshooting labs - broken BGP configurations

### DISCUSSION THREAD

**Discussion by userreeducationtool** [bgplab/bgplab#9] [Ideas]
First GitHub comment, hope I'm doing this right!

Have there been any thoughts / discussions / ideas around some labs that involve troubleshooting and fixing broken BGP configurations?  I'm thinking along the lines of situations I've seen in other vendors training classes and lab exercises where you're presented with a configuration, the customer wants X to happen, but Y is happening now.  You need to reconfigure (sometimes with guidelines, this device can be changed, this one cannot, etc.) the lab topology as needed to work properly.

**Comment by ipspace** [comment]
Interesting idea, thanks a million. Will think about it.
